### PR TITLE
updates libraries_locations to remove ILL links from availability

### LINKED ISSUE #1124: No Recalls Deploy: Update libraries_locations.json to empty ILL links  and rename fields
By emptying the Request via ILL location group, we can avoid generating the ILLiad link in Availability (much like with the [reserves_scan location group](https://github.com/psu-libraries/psulib_blacklight/blob/main/app/javascript/availability/libraries_locations.json#L798)). Also the locations will need renaming in the main section to remove direction to request via ILL.

- [x] empty the `request_via_ill` location group.
- [x] replace the _individual fields_ in "locations" section using the affected fields list below. Note that though pairs may be adjacent, these will not be in a block so they're coded below as individual fields.
- [ ] the two steps above should mean that the Availability section will no longer generate the ILL link code. This should be verified and a follow-up ticket should be made to remove the code if we don't have time to get to it before MVP no recalls release. Or we could leave in place like reserves_scan, but we should at least discuss.

## Affected Fields List

`"BINDERY-DN": "At bindery",`
`"BINDERY-HY": "At bindery",`
`"BINDERY": "At bindery",`
`"BINDPRP-DN": "At bindery",`
`"BTAA": "Offsite at BTAA facility",`
`"CANCELED": "Copy unavailable",`
`"CHECKEDOUT": "Checked out",`
`"DARCHIVE-P": "Copy unavailable",`
`"DEPTCHARGE": "Copy unavailable",`
`"HARSHBARGR": "Copy unavailable",`
`"ILL": "Checked out",`
`"ILLEND": "Checked out",`
`"LOST": "Copy unavailable",`
`"MISSING": "Copy unavailable",`
`"REPAIR": "Being repaired",`

### PR DESCRIPTION
**Pull request by Smullz622**
fixes #1124 

This takes care of the code changes for the issue, but it still needs to be discussed whether we want to create a follow up ticket to remove `request_via_ill` entirely or leave it blank like `reserves_scan`

### CODE REVIEW

File: app/javascript/availability/libraries_locations.json [hunk 1]
```diff
@@ -568,7 +568,7 @@
     "REF-WB": "Wilkes-Barre - Reference Collection - In-Library Use",
     "REF-WS": "Scranton - Reference Collection - In-Library Use",
     "REF-YK": "York - Reference Collection - In-Library Us",
-    "REPAIR": "Copy unavailable, request via Interlibrary Loan",
+    "REPAIR": "Copy unavailable",
```
Comment (banukutlu) [inline app/javascript/availability/libraries_locations.json:571]: @Smullz622 in the issue this location looks like this
```
"REPAIR": "Being repaired",
```

**Review by banukutlu — CHANGES_REQUESTED** [review]

File: app/javascript/availability/libraries_locations.json [hunk 2]
```diff
@@ -232,12 +232,12 @@
...
     "ILL-BK": "Berks - ILL Office",
     "ILL-BR": "Beaver - ILL Office",
-    "ILLEND": "Copy unavailable, request via Interlibrary Loan",
+    "ILLEND": "Copy unavailable",
     "ILL-HB": "Harrisburg - Interlibrary Loan Office - 2nd Floor",
     "ILL-HY": "Hershey (College of Medicine), ILL Office",
     "ILL-SL": "Schuylkill - ILL Office",
     "ILL-SV": "Shenango - ILL Office",
-    "ILL": "Copy unavailable, request via Interlibrary Loan",
+    "ILL": "Copy unavailable",
```
Comment (banukutlu) [inline app/javascript/availability/libraries_locations.json:240]: @Smullz622 Looks like this needs to be checked out

```
"ILL": "Checked out",
"ILLEND": "Checked out",
```

File: app/javascript/availability/libraries_locations.json [hunk 3]
```diff
@@ -232,12 +232,12 @@
     "IDEALAB-BK": "Berks - IDEA Lab (Instructional Design Lab)",
     "ILL-BK": "Berks - ILL Office",
     "ILL-BR": "Beaver - ILL Office",
-    "ILLEND": "Copy unavailable, request via Interlibrary Loan",
+    "ILLEND": "Copy unavailable",
```
Comment (banukutlu) [inline app/javascript/availability/libraries_locations.json:235]: @Smullz622 Looks like this needs to be checked out

```
"ILL": "Checked out",
"ILLEND": "Checked out",
```

**Review by banukutlu — CHANGES_REQUESTED** [review]

File: app/javascript/availability/libraries_locations.json (hunk 3 above)
Comment (Smullz622) [reply app/javascript/availability/libraries_locations.json]: My mistake, I think I've got them all now. Thanks!

Comment (banukutlu) [reply app/javascript/availability/libraries_locations.json]: no worries, thanks for the fix :)

File: spec/javascript/availability/components/special_request_link.jsx.spec.js [hunk 4]
```diff
@@ -182,42 +181,6 @@ describe('when locationText is not sent to SpecialRequestLink', () => {
...
-
-      await testLink(
-        getByRole,
-        container,
-        'Copy unavailable, request via Interlibrary Loan',
-        href
-      );
-    });
-
-    test('renders a reserves scan link', async () => {
```
Comment (banukutlu) [inline spec/javascript/availability/components/special_request_link.jsx.spec.js:200]: @Smullz622 do we need to remove this test?

**Review by banukutlu — APPROVED** [review]
👍🏻

**Pull request merged**